app/models/showdate.rb

```diff
@@ -34,6 +34,7 @@ def customers
   validates_length_of :description, :maximum => 32, :allow_nil => true
   
   attr_accessible :thedate, :end_advance_sales, :max_advance_sales, :description, :show_id, :seatmap_id
+  attr_accessible :valid_vouchers
```

Comment:
it shouldn't be possible to mass-assign this attribute. this line doesn't belong here.  i think you have attr_accessible confused with something else.  please google it.

Reply:
It seems that there are tons of not-internally-reviewed code in #b743bf5, I will do it right now.